src/Gui/View3DInventorViewer.cpp

```diff
@@ -1171,6 +1171,36 @@ void View3DInventorViewer::setNaviCubeCorner(int c)
         naviCube->setCorner(static_cast<NaviCube::Corner>(c));
 }
 
+void View3DInventorViewer::setNaviCubeSize(int size)
```

Comment:
All these methods are not really needed. The View3DInventor class can directly access the NaviCube and then invoke the corresponding methods directly.

Reply:
Thanks, now fixed.